paddle/fluid/operators/batch_norm_op.cc

```diff
@@ -598,36 +598,13 @@ std::unique_ptr<framework::OpDesc> BatchNormGradMaker::Apply() const {
   return std::unique_ptr<framework::OpDesc>(op);
 }
 
-class BatchNormInplaceInToOut : public framework::InplaceOpInference {
```

Comment:
Delete but no DECLARE_INPLACE_OP_INFERER?

Reply:
This is cleaning unused codes.